generator/src/main/java/com/scottlogic/deg/generator/validators/VisualisationConfigValidator.java

```diff
@@ -0,0 +1,60 @@
+package com.scottlogic.deg.generator.validators;
+
+import com.google.inject.Inject;
+import com.scottlogic.deg.generator.outputs.targets.FileOutputTarget;
+import com.scottlogic.deg.generator.outputs.targets.OutputTarget;
+import com.scottlogic.deg.generator.utils.FileUtils;
+import com.scottlogic.deg.generator.visualisation.VisualisationConfigSource;
+
+import java.util.ArrayList;
+
+/**
+ * Class used to determine whether the command line options are valid for visualisation
+ */
+public class VisualisationConfigValidator {
+
+    private final FileUtils fileUtils;
```

Comment:
I know I introduced FileUtils but I'm wondering if we should rename it? FileUtils is a super useful class from `org.apache.commons.io`

Reply:
As this is being updated in other issues, I would rather not rename it here.